boottool.sh

```diff
@@ -0,0 +1,311 @@
+#! /bin/env bash
+echo $BASH_VERSION
+echo "number of args = $#"
+TOKEN="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"
+DEFAULT_OWNER="ZOSOpenTools"
+OWNERNAME=""
+REPO=""
+RELEASENAME=""
+CURL_POST="curl -sw "%{http_code}" -X POST -H \"Accept: application/vnd.github+json\""
+CURL_REPO_SUCCESS="200"
+DELETE_RELEASE_CODE="204"
+CRT_UPLOAD_REL_SUCCESS="201"
+
+processOptions()
+{
+  args=$*
+  while [[ $# -gt 0 ]]; do
+    case $1 in
+        "-uname")
+          shift
+          OWNERNAME="${1}"
+          ;;
+        "-release")
+          shift
+          RELEASENAME="${1}"
+          ;;
+        "-repo")
```

Comment:
If it’s a multi-character option it is usually 2 dashes eg —repo

Reply:
Oh Ok! Sure i have modified it.